src/main/java/Duke.java

```diff
@@ -1,10 +1,244 @@
+                    }
+                    lists.add(deadline);
+                } else if (task[0].equals("E")) {
+                    Task event = new Event(task[4], LocalDateTime.parse(task[6]));
+                    if (task[2].equals("✓")) {
+                        event.markDone();
+                    }
+                    lists.add(event);
+                } else {
+                    continue;
+                }
+            }
+            Scanner sc = new Scanner(System.in);
+            String logo = "      ___                __     \n"
+                    + "     |  _ \\   ____      |  |    \n"
+                    + "     | |_| | / __  \\  __|  |__  \n"
+                    + "     |  _ / | |  |  ||__    __| \n"
+                    + "     | |    | |_/   |   |  |    \n"
+                    + "     |_|     \\___/|_|   |__|    \n";
+            String[] greetingTexts = {"Hello! I'm Pat", "What can I do for you?"};
+            Response greeting = new Response(greetingTexts);
+            System.out.println("     Hello from\n" + logo);
+            System.out.println(greeting.getResponse());
+            while (sc.hasNext()) {
+                String received = sc.nextLine();
+                if (received.equals("bye")) {
+                    String[] byeMessage = {"Bye. Hope to see you again soon!"};
+                    Response bye = new Response(byeMessage);
+                    System.out.println(bye.getResponse());
+                    break;
+                } else if (received.equals("list")) {
+                    Response list = new Response(lists.toArray(new Task[0]));
+                    System.out.println(list.getResponse());
+                } else {
+                    String[] test = received.split(" ");
+                    if (test[0].equals("done")) {
+                        try {
+                            if (lists.size() == 0) {
+                                throw new IndexOutOfBoundsException();
+                            }
+                            if (test.length == 1) {
+                                throw new MissingNumberException();
+                            }
+                            int idx = Integer.parseInt(test[1]) - 1;
+                            Task target = lists.get(idx);
+                            target.markDone();
+                            Response msg = new Response(new String[]{"Nice! I've marked this task as done:", "  " + target});
+                            System.out.println(msg.getResponse());
+                        } catch (IndexOutOfBoundsException e) {
+                            Response msg = new Response(new String[]{"The number does not exist!"});
+                            System.out.println(msg.getResponse());
+                        } catch (MissingNumberException e) {
+                            Response msg = new Response(new String[]{"Please select the task that you want to mark done!"});
+                        }
```

Comment:
Perhaps you can print out the response message to the user on line 86 like u did for line 83?